typing/btype.mli

```diff
@@ -94,6 +96,13 @@ val proxy: type_expr -> type_expr
         (* Return the proxy representative of the type: either itself
            or a row variable *)
 
+(* Poly types. *)
+
+(* These three functions can only be called on [Tpoly] nodes. *)
+val is_mono : type_expr -> bool
```

Comment:
I don't know the local customs on this, but I would expect these function names to be louder in that they're very partial. Maybe `tpoly_is_mono`, `tpoly_get_mono`, `tpoly_get_poly`?

Reply:
Those names are better, it now uses those.